environs/__init__.py

```diff
@@ -250,9 +251,15 @@ class Env:
     dj_email_url = _func2method(_dj_email_url_parser, "dj_email_url")
     dj_cache_url = _func2method(_dj_cache_url_parser, "dj_cache_url")
 
-    def __init__(self, *, eager: _BoolType = True):
+    def __init__(
+        self, *, eager: _BoolType = True, allow_proxy: _BoolType = True, expand_vars: _BoolType = False
+    ):
         self.eager = eager
         self._sealed = False  # type: bool
+        self.allow_proxy = allow_proxy
+        self.expand_vars = expand_vars
+        # TODO: disallow setting both allow_proxy and expand_vars?
```

Comment:
suggestion: Since this is non-breaking, it might be a good idea to do this now.

If `allow_proxy` and `expand_vars` are both `True`, it's most likely due to user error, I'd think.